sections/errorhandling/usematurelogger.md

```diff
@@ -2,43 +2,19 @@
 
 ### One Paragraph Explainer
 
-We all love console.log but obviously, a reputable and persistent logger like [Winston][winston] (highly popular) or [Pino][pino] (the new kid in town which is focused on performance) is mandatory for serious projects. A set of practices and tools will help to reason about errors much quicker – (1) log frequently using different levels (debug, info, error), (2) when logging, provide contextual information as JSON objects, see example below. (3) Watch and filter logs using a log querying API (built-in in most loggers) or a log viewer software. (4) Expose and curate log statement for the operation team using operational intelligence tools like Splunk.
+We all love console.log but obviously, a reputable and persistent logger like [Pino][pino] (a [newer option focused on performance][dropdiscussion]) is mandatory for serious projects. 

```

Comment:
I would remove `[dropdiscussion]` and add a line below that says something like: `If you are wondering why winston is not present here see #684 `

Reply:
Agreed - adding 

For more information on why traditional favorites (e.g., Winston) may not be included in the current list of recommended best practices, please see #684.

Reply:
Looks good! From [what I see](https://github.com/denisecase/nodebestpractices/blob/master/sections/errorhandling/usematurelogger.md), It seems that github does not interpret the # issue number in markdown files. Would you mind transforming it into a link?
After that it's good to merge for me :)

Reply:
Thanks @kevynb - I've enabled the link in the text.. it's relative, so it won't find 684 while over in my fork, but I think it'll work once brought back home.

Reply:
Ah I understand. Thanks for explaining to me!